Blish HUD/Controls/Scrollbar.cs

```diff
@@ -86,29 +86,33 @@ public class Scrollbar:Control {
 
         private int TrackLength => (_size.Y - _textureUpArrow.Height - _textureDownArrow.Height);
 
-        private bool Scrolling = false;
-        private int ScrollingOffset = 0;
+        private bool _scrolling       = false;
+        private int  _scrollingOffset = 0;
 
-        private Rectangle UpArrowBounds;
-        private Rectangle DownArrowBounds;
-        private Rectangle BarBounds;
-        private Rectangle TrackBounds;
+        private Rectangle _upArrowBounds;
+        private Rectangle _downArrowBounds;
+        private Rectangle _barBounds;
+        private Rectangle _trackBounds;
 
-        public Scrollbar(Container container) : base() {
+        public Scrollbar(Container container) {
             _associatedContainer = container;
 
-            UpArrowBounds = Rectangle.Empty;
-            DownArrowBounds = Rectangle.Empty;
-            BarBounds = Rectangle.Empty;
-            TrackBounds = Rectangle.Empty;
+            _upArrowBounds   = Rectangle.Empty;
+            _downArrowBounds = Rectangle.Empty;
+            _barBounds       = Rectangle.Empty;
+            _trackBounds     = Rectangle.Empty;
 
             this.Width = CONTROL_WIDTH;
 
-            Input.Mouse.LeftMouseButtonReleased += delegate { if (Scrolling) { Scrolling = false; /* Invalidate(); */ } };
+            Input.Mouse.LeftMouseButtonReleased += MouseOnLeftMouseButtonReleased;
 
             _associatedContainer.MouseWheelScrolled += HandleWheelScroll;
         }
 
+        private void MouseOnLeftMouseButtonReleased(object sender, MouseEventArgs e) {
+            _scrolling = false;
+        }
+
```

Comment:
Sometimes you use `Input_EventInPascalCase` and sometimes you use `InputEventInPascalCase`. Since this is a cleanup pr, it would be a great opportunity to get this straight.

Reply:
Do you mind linking an example of both so I can ensure I'm fully on the same page.  I agree that this is likely a good time to get whatever it is consistent.  🙂

Reply:
i sent you all occurences of both on discord
here they are again:
https://cdn.discordapp.com/attachments/558957192493924364/694437814951215144/message.txt
https://cdn.discordapp.com/attachments/558957192493924364/694438320448602142/message.txt

Reply:
i just saw that the latter one is missing 3 functions, because my regex didnt account for numbers in the function name

Reply:
Thank you for sending those!  That is very odd.  I haven't noticed that.  Looks like VS tries to auto-complete it two different ways.  Likely a consequence of having a style specified in ReSharper that maybe doesn't match the Visual Studio default.

https://gfycat.com/wealthydecisivearrowworm

You can see the first tooltip show it with an `_` and the dropdown from the IntelliSense showing it without.

Reply:
Okay, the `_` is apparently the default in the VS provided method and ReSharper was providing its own.  After some review, I've opted to go with the convention ReSharper uses.  Fixed the ones you linked in 3d34091 & 949e518.